array.ipynb

```diff
@@ -130,6 +130,15 @@
    "source": [
     "%time y.sum().compute()"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Further Reading \n",
+    "\n",
+    "A more in-depth guide to working with Dask arrays can be found in the [dask tutorial](https://github.com/dask/dask-tutorial), notebook 03."
```

Comment:
We might want to avoid enumerating the notebook here.  I suspect that a reader will be able to find the array notebook, given the name.  I wouldn't be surprised if the notebook number changes in a year or so, and it would be nice not to have to update these references when that happens.

Reply:
Should we also include the documentation for each section here?  That would also convert this text into a bulleted list, which I think tends to increase engagement a bit.

Reply:
> We might want to avoid enumerating the notebook here

I agree, bu I see this as a medium-term only, until the tutorial is also (mostly) binderised, and then we can link directly to the notebook(s) in question.